README.md

```diff
@@ -44,7 +44,13 @@ west config build.sysbuild True
 To build the application, run the following command:
 
 ```shell
-west build -b thingy91x/nrf9151/ns app
+west build -b thingy91x/nrf9151/ns --sysbuild app
```

Comment:
Is this step nessecary when we instruct the user to call:

```
west config build.sysbuild True
```

Reply:
you're right, I removed it.